modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLFileEntryConfiguration.java

```diff
@@ -9,24 +9,23 @@
 
 import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
 
-import org.osgi.annotation.versioning.ProviderType;
-
 /**
  * @author Drew Brokke
  */
-@ExtendedObjectClassDefinition(category = "documents-and-media")
+@ExtendedObjectClassDefinition(
+	category = "documents-and-media", generateUI = false,
+	scope = ExtendedObjectClassDefinition.Scope.GROUP
+)
 @Meta.OCD(
 	id = "com.liferay.document.library.configuration.DLFileEntryConfiguration",
 	localization = "content/Language", name = "dl-file-entry-configuration-name"
 )
-@ProviderType
 public interface DLFileEntryConfiguration {
 
-	@Meta.AD(
-		deflt = "104857600",
-		description = "previewable-processor-max-size-help",
-		name = "previewable-processor-max-size", required = false
-	)
+	@Meta.AD(deflt = "20971520", name = "maximum-file-size", required = false)
```

Comment:
Why the change in the default value?

Reply:
This can break installations for people that didn't change the original value, as operations that succeeded before now will fail.

Reply:
As we have discussed in meeting this default value change is needed. We have to state it clearly in Breaking changes